.ideainspect

```diff
@@ -21,3 +15,10 @@ resultdir: target/inspection-results
 
 # IDEA project root directory containing the ".idea" directory
 # rootdir: .
+
+# Library folder containing idea.properties More info at:
+# http://tools.android.com/tech-docs/configuration
+# lib: ~/Library/Preferences/AndroidStudio2.1/
+
+# Custom IDEA Scope name
```

Comment:
A few more notes how this name relates to the name you entered in the IDEA UI (ie. "My Fancy!1@ Scope") would be helpful. I assume it is the file name located in `.idea/scopes` without the `.xml` suffix?